buildpacks/hello-world/bin/build

```diff
@@ -18,6 +18,15 @@ if compgen -G "${env_dir}/*" > /dev/null; then
 fi
 export | sed 's/^/       /'
 
+# VOLUMES
+VOLUME_FILE=/platform/volume/volume_contents.txt
```

Comment:
channeling sclevine here, but I think he would say to make this lowercase unless you are going to export it